cmd/vic-machine/inspect/inspect.go

```diff
@@ -99,21 +118,24 @@ func (i *Inspect) processParams() error {
 	return nil
 }
 
-func (i *Inspect) Run(clic *cli.Context) (err error) {
+func (i *Inspect) run(clic *cli.Context, cmd command) (err error) {
 	// urfave/cli will print out exit in error handling, so no more information in main method can be printed out.
 	defer func() {
 		err = common.LogErrorIfAny(clic, err)
 	}()
 
-	if err = i.processParams(); err != nil {
-		return err
-	}
-
-	if i.Debug.Debug != nil && *i.Debug.Debug > 0 {
+	if i.Output != "" {
```

Comment:
here, might compare the value with "raw", if not "raw", take it as "", with warning message